docs: show step arrows (#705)
appsmithorg/appsmith-docs: website/docs/reference/widgets/currency-input.md

| **Placeholder**           | Sets the placeholder text within the input box. Use to show a hint or example value to the user.                                                                                                       |
| **Tooltip**               | Sets a tooltip that appears when the user hovers over the widget with the mouse. Use this to provide hints or extra information to the user.                                                           |
| **Required**              | Sets whether the checkbox is a mandatory field. When the checkbox is within a Form widget, that Form's submit button will be automatically disabled until the Checkbox is checked.                     |

> | **Show step arrows**              | Control's visibility of step arrows. When turned off: The step arrows to increment or decrement the values in the widget will be hidden.                                                                        |

Reviewer: ```suggestion
| **Show step arrows**              | Controls visibility of step arrows. Turning this toggle off hides the step arrows to increment or decrement the values in the widget.                                                                        |
```
